spec/helpers/submission_helper_spec.rb

```diff
@@ -37,4 +37,30 @@
       end
     end
   end
+
+  describe '#get_file_info' do
+    let(:assignment) { create(:assignment) }
+    let(:grouping) { create(:grouping, assignment: assignment) }
+    let(:file_name) { 'test.zip' }
+    before do
+      grouping.group.access_repo do |repo|
```

Comment:
use `grouping.access_repo` directly

Reply:
Actually, you don't need to actually submit a file to the repo to test this method. You can create a RevisionFile object directly for use in your tests. That way, this test will fail only if there is an issue with the `get_file_info` method, not if there is an issue with repo access/setup.